csv/header.go

```diff
@@ -19,6 +21,11 @@ func NewHeader(header []string) *Header {
 
 // DimensionOffset returns the number of columns before the start of the dimension columns.
 func (header Header) DimensionOffset() (int, error) {
+	// return 0, nil
+	log.Info("Header", log.Data{"header len": len(header.row[0])})
```

Comment:
remove commented out line, consider assigning len(header.row[0]) to a var, and "header len" should be "header_len"